docs/release/3.4/release-3-4-39.md

```diff
@@ -0,0 +1,220 @@
+OSG Software Release 3.4.39
+===========================
+
+**Release Date**: 2019-11-14
+**Supported OS Versions:** EL7, EL6
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   [Frontier-Squid 4.9-2.1](http://frontier.cern.ch/dist/rpms/frontier-squidRELEASE_NOTES): [Fixed security vulnerabilities](https://opensciencegrid.org/security/vulns/OSG-SEC-2019-11-11-Vulnerability-in-Squid-UPDATE/)
+    -   [Heap Overflow issue in URN processing](http://www.squid-cache.org/Advisories/SQUID-2019_7.txt)
+    -   [Multiple issues in URI processing](http://www.squid-cache.org/Advisories/SQUID-2019_8.txt)
+-   [XRootD 4.11.0](https://github.com/xrootd/xrootd/blob/v4.11.0/docs/ReleaseNotes.txt): Support for multi-VO credentials
+-   [BLAHP 1.18.45](https://github.com/htcondor/BLAH/releases/tag/v1.18.44)
+    -   Add Slurm SystemCpu accounting when job completes
+    -   Add support for decimal seconds fields from sacct
+    -   Move local submit attributes scripts from spec to sources
+    -   Pull in various updates to Slurm scripts from HTCondor's BLAHP
+        -   Remove superfluous debugging statements
+        -   Add BLAHP options for project name (-A, BatchProject) and runtime limit (-t, BatchRuntime).
```

Comment:
It should be noted that the `BatchProject` and `BatchRuntime` are HTCondor job attributes